contracts/script/HotShot.s.sol

```diff
@@ -2,14 +2,38 @@
 pragma solidity ^0.8.0;
 
 import "forge-std/Script.sol";
-import { HotShot } from "../src/HotShot.sol";
+import { LightClient as LC } from "../src/LightClient.sol";
+import { LightClient } from "../src/LightClient.sol";
 
 contract DeployHotShot is Script {
     function run() external {
         string memory seedPhrase = vm.envString("MNEMONIC");
         uint256 privateKey = vm.deriveKey(seedPhrase, 0);
         vm.startBroadcast(privateKey);
-        new HotShot();
+
+        //// LightClient contract deployment
+
+        // For Decaf there will be only one epoch
+        uint32 blocksPerEpoch = type(uint32).max;
+
+        // TODO change when we start using epochs. See
+        // https://github.com/EspressoSystems/espresso-sequencer/issues/940
+        uint256 stakeTableCapacity = 10;
```

Comment:
this comment is a little misleading, technically we don't need this at all. here we only need it since we are generating a "mock genesis", 
in actual production deployment script, this will be removed (even when we are using epochs)

this value will only be used in `StakeTable.s.sol`, not in this file

Reply:
I see, misleading comment removed in ae07794ae91771bf150b36f95348da0e34f8743e.